### PR TITLE
github actions to make tar files for releases of automatic-actions

### PR DESCRIPTION
**Pull request by sbe-arg**
makes a release just for automatic-actions with the tar files required to open prs in https://github.com/marvinpinto/action-automatic-releases

### CODE REVIEW

**Review by Enase — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml
Comment (Enase) [reply .github/workflows/release.yml:39]: Just run: `yarn run lintfix:eslint` and `yarn run lintfix:prettier` from repo root in order to fix [CI issues](https://github.com/marvinpinto/actions/actions/runs/5396388402/jobs/9799962017?pr=675)

**Comment by sbe-arg** [comment]
@Enase mi energy and motivation to fix this is low.
we wither disable commit linting entirely or I won't do further testing on this. 

Too much wiggle around.
This structure was poorly designed. where once you merge a PR future checks fail. 
The CI should block merging a PR with the wrong commit message or should have autofixed commit messages so linting does into break after each pr.

Ill leave it with you. 
You can open a fresh pr with the changes you consider Ill just approve your prs.

I have moved most of my repos to this action I made this week https://github.com/marketplace/actions/simple-tags-and-releases its 20 lines of code and does not require much support. 
Offers the minimal functionality I need with uploading single files and passing and creating tags from a source file.

**Pull request closed without merging**
